tests/integration/goldens/asset/samples/generated_samples/cloudasset_generated_asset_v1_asset_service_create_feed_sync.py

```diff
@@ -34,7 +34,13 @@ def sample_create_feed():
     client = asset_v1.AssetServiceClient()
 
     # Initialize request argument(s)
+    feed = asset_v1.Feed()
+    feed.name = "name_value"
+
     request = asset_v1.CreateFeedRequest(
+        parent="parent_value",
+        feed_id="feed_id_value",
+        feed=feed,
     )
 
```

Comment:
Okay I know it's not part of this generation, but I noticed .format - are we supporting less than Python 3.6 and that's why no f-strings?

Reply:
Nope, no support for less than 3.6. I think the `.format` is from a previous iteration that _was_ supposed to support older Python versions. Opened https://github.com/googleapis/gapic-generator-python/issues/960 to track switching to f-strings.

Reply:
@busunkim96 is correct. Earlier in the dev process for the python microgenerator, there was an expected need to support 3.5.